src/utils/formatCurrencyAmount.ts

```diff
@@ -0,0 +1,31 @@
+import { isInteger } from 'lodash'
+import { V2V3CurrencyOption } from 'models/v2v3/currencyOption'
+import { V2V3_CURRENCY_ETH, V2V3_CURRENCY_USD } from './v2v3/currency'
+
+const formatAmount = (amount: number) => {
```

Comment:
Could this arg be `number | string` or something else https://lodash.com/docs/4.17.15#isInteger.

Reply:
potentially but lets wait till we have a usecase I reckon